.github/workflows/ci.yml

```diff
@@ -102,6 +102,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.50.1
```

Comment:
@diligentis The latest version is 1.51.0. When running with the latest, the lint fails saying "var 'np' unused". However, the error does not state the file or line which the lint fails. I turned on debugging and verbose output and that did not change the output of the lint.

Reply:
https://github.com/golangci/golangci-lint/releases

Reply:
Sounds reasonable for the time being until we can get that figured out. Thanks!